Makefile

```diff
@@ -78,7 +78,7 @@ HELM_CHART_LINT_ARGS_$(PACKAGE_NAME) = --set nameOverride='',imagePullSecrets=''
 # Due to the way that the shared build logic works, images should
 # all be in folders at the same level (no additional levels of nesting).
 
-REGISTRY_ORGS ?= docker.io/upbound
+REGISTRY_ORGS ?= xpkg.upbound.io/upbound
```

Comment:
Will also need to update location in https://github.com/upbound/universal-crossplane/blob/2b692a94243814661df73be4ca9212f244c4d824/cluster/charts/universal-crossplane/values.yaml.tmpl#L157

Reply:
Done, thanks!